integration_tests/mockApis/componentsApi.ts

```diff
@@ -0,0 +1,17 @@
+import { stubFor } from './wiremock'
+
+const stubComponents = () => {
+  return stubFor({
+    request: {
+      method: 'GET',
+      url: '/components/components?component=header&component=footer',
+    },
+    response: {
+      status: 500,
+    },
+  })
+}
+
```

Comment:
do we really need this stub?

Reply:
Functionally no, but it clears up the console output during e2e tests and makes debugging easier imo